src/complex_matmul.jl

```diff
@@ -25,6 +25,133 @@ for AT in [:AbstractVector, :AbstractMatrix]  # to avoid ambiguity error
             _C
         end
 
+        function _matmul_v2!(_C::$AT{Complex{T}}, _A::AbstractMatrix{Complex{U}}, _B::$AT{Complex{V}}, 
+                            α=One(), β=Zero(), nthread::Nothing=nothing, MKN=nothing, contig_axis=nothing) where {T,U,V}
+            # C, A, B = map(real_rep, (_C, _A, _B))
+            C = reinterpret(T, _C)
+            A = reinterpret(T, _A)
+            B = real_rep(_B)
+
+            η_bool = ifelse(ArrayInterface.is_lazy_conjugate(_A), StaticInt(-1), StaticInt(1))
+            θ_bool = ifelse(ArrayInterface.is_lazy_conjugate(_B), StaticInt(-1), StaticInt(1))
+            (+ᶻ, -ᶻ) = ifelse(ArrayInterface.is_lazy_conjugate(_C), (-, +), (+, -))
+            # ηθ = η*θ
+
+            signs = Vec(ntuple(x -> ifelse(iseven(x), -one(T), one(T)), pick_vector_width(Float64))...)
+            if !η_bool & !θ_bool
+                cmatmul_ab(C, A, B)
+
```

Comment:
```suggestion
                end
```
maybe?
Not sure what's going on there.

Reply:
Yeah this is all still very WIP. I'll work on this again tomorrow (hard maybe), just wanted to get this pushed since was switching around on machines.